crates/node/src/commands/run.rs

```diff
@@ -69,17 +72,16 @@ pub fn run_node(mut cli: Cli) -> Result<()> {
         override_dev_environment(&mut cli.run);
     }
     let runner = cli.create_runner(&cli.run.base)?;
-    let data_path = &runner.config().data_path;
 
     let da_config: Option<(DaLayer, PathBuf)> = match cli.run.da_layer {
         Some(da_layer) => {
-            let da_path = data_path.join("da-config.json");
-            if !da_path.exists() {
-                log::info!("{} does not contain DA config", da_path.display());
+            let da_conf = PathBuf::from(cli.run.da_conf.expect("no da config provided"));
```

Comment:
`.expect("clap requires `da_conf` when  `da_layer` is present"));`

Reply:
already handled by clap argroup

Reply:
yep. I meant "modify the error message to make it explicitly state why it is safe to unwrap here"